From d70611ff5199156e97c817b78dad7adf98b2e3c7 Mon Sep 17 00:00:00 2001
From: WangYekun <849907942@qq.com>
Date: Wed, 4 Dec 2019 11:28:36 +0800
Subject: [PATCH] fix: delete order-backed

---
 order-backend | 1 -
 1 file changed, 1 deletion(-)
 delete mode 160000 order-backend

diff --git a/order-backend b/order-backend
deleted file mode 160000
index 85f40bb..0000000
--- a/order-backend
+++ /dev/null
@@ -1 +0,0 @@
-Subproject commit 85f40bb83c9cc5af13d77780de2800655f033039
-- 
2.9.0.windows.1

